examples/embedders/gtfs2vec_embedder.ipynb

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
<p>Naming here again looks a bit confusing. You already have features (features_gdf), why do you need to use an embedder to get them? Maybe call it something like "Get aggregations without using a neural network embedder? It could be similar to the <code>skip_embedding</code> parameter name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/srai-lab/srai/pull/169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>